src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishASelfContainedApp.cs

```diff
@@ -196,6 +197,13 @@ public void It_publishes_runtime_pack_resources()
                 "zh-Hans/PresentationCore.resources.dll",
                 "zh-Hant/PresentationCore.resources.dll",
             });
+
+            using (var depsJsonFileStream = File.OpenRead(Path.Combine(output.FullName, $"{testProject.Name}.deps.json")))
+            {
+                var dependencyContext = new DependencyContextJsonReader().Read(depsJsonFileStream);
+                dependencyContext.Should()
+                    .OnlyHaveRuntimePacksWithFrameworkNameProperties();
+            }
```

Comment:
Unless we have that test somewhere else it would be good to validate that there are at least some runtime packs written in the `.deps.json`.